.github/workflows/lint.yaml

```diff
@@ -29,4 +29,6 @@ jobs:
         with:
           node-version: 22
       - run: "npm --prefix new-log-viewer/ clean-install"
+      - run: "npm --prefix new-log-viewer/ run lint:css-ci"
       - run: "npm --prefix new-log-viewer/ run lint:check"
```

Comment:
Sorry I think this was supposed to be `lint:js-check`.
```suggestion
      - run: "npm --prefix new-log-viewer/ run lint:js-check"
```

Reply:
yeah it turn out that no changes are needed in the workflow file after the scripts are simplified